kustomize/commands/edit/add/flagsandargs_test.go

```diff
@@ -79,9 +79,9 @@ func TestDataConfigValidation_Flags(t *testing.T) {
 	for _, test := range tests {
 		err := test.fa.Validate([]string{"name"})
 		if test.shouldFail {
-			assert.Error(t, err)
+			require.Error(t, err)
```

Comment:
nit: change the above `asserts` to `require` as well, so that we are not importing two packages when we only need one

Reply:
Replaced!